src/js/main.js

```diff
@@ -290,19 +290,26 @@ setViewportValues();
 
 // Flip clock for instances
 const flip = document.querySelector(".flip");
-const flipText = document.querySelector(".flip-text");
+const flipTexts = document.querySelectorAll(".flip-text");
```

Comment:
If you keep it all in the same `div`, you can restore this code to work with just that single element.